src/tumboss.js

```diff
@@ -45,23 +48,40 @@ controller.hears( 'tumboss', 'direct_message,direct_mention,mention,ambient', fu
 
 controller.hears( [ '\\bessen\\b', 'mensa', 'mittagessen', 'hunger', 'kohldampf' ], 'direct_message,direct_mention,mention,ambient', function( bot, message ) {
 	console.log( 'Starting request for Mensa plan' );
```

Comment:
This comment should actually be moved down into the if because only then the request gets started.